packages/rule-meta-theme-color/src/rule.ts

```diff
@@ -0,0 +1,188 @@
+/**
+ * @fileoverview Check if a single `<meta name="theme-color">` is
+ * specified in the `<head>`.
+ */
+
+/*
+ * ------------------------------------------------------------------------------
+ * Requirements
+ * ------------------------------------------------------------------------------
+ */
+
+import { isSupported } from 'caniuse-api';
+import { get as parseColor } from 'color-string';
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+/* eslint-disable no-unused-vars */
+import {
+    ElementFound,
+    IAsyncHTMLDocument,
+    IAsyncHTMLElement,
+    IRule,
+    RuleMetadata,
+    TraverseEnd
+} from 'sonarwhal/dist/src/lib/types';
+/* eslint-enable no-unused-vars */
+import {
+    isHTMLDocument,
+    normalizeString
+} from 'sonarwhal/dist/src/lib/utils/misc';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class MetaThemeColorRule implements IRule {
+
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.pwa,
+            description: `Require a 'theme-color' meta tag`
+        },
+        id: 'meta-theme-color',
+        schema: [],
+        scope: RuleScope.any
+    }
+
+    public constructor(context: RuleContext) {
+
+        let bodyElementWasReached: boolean = false;
+        let firstThemeColorMetaTag: IAsyncHTMLElement;
+
+        const checkIfThemeColorMetaTagWasSpecified = async (event: TraverseEnd) => {
+            const { resource } = event;
+
+            /*
+             * TODO: Remove this when `traverse::end` is
```

Comment:
Did you open an issue about it? We should link to it.

Reply:
@molant It's basically the same issue as https://github.com/sonarwhal/sonarwhal/issues/982.

Reply:
Can you link to it in the code?

Reply:
Done.